taskcluster/ci/split-mono-src/kind.yml

```diff
@@ -66,17 +62,12 @@ task-defaults:
             - -c
               # TODO: move number of chunks elsewhere
```

Comment:
Question (aside, non-blocking): Is this TODO still valid? I'm not sure why it's a TODO.

Reply:
It's a question for @bhearsum

Reply:
Ah - no, this is outdated now. (It used to be hardcoded in this file.)